tests/pytorch/iris.py

```diff
@@ -36,7 +36,7 @@ def training_step(self, batch, batch_idx):
         x, y = batch
         logits = self.forward(x)
         loss = self.cross_entropy_loss(logits, y)
-        self.train_acc(logits, y)
+        self.train_acc(torch.argmax(logits, dim=1), y)
```

Comment:
Fix for this errro:

https://github.com/mlflow/mlflow/runs/1880410847?check_suite_focus=true#step:8:1930

Reply:
This error means:


```python
preds = [
  # ===> C dimension
  [0.2, 0.3, 0.5]  # good because this row sums up to 1
  [0.1, 0.2, 0.3]  # bad because this row doesn't
]
```